recipes/mlconjug3/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "mlconjug3" %}
+{% set version = "3.7.15" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mlconjug3-{{ version }}.tar.gz
+  sha256: 442237d9095e55b0d2491e33d768ff0f837a4f68ff5491d7e2e8ea5e9e182b62
+
+build:
+  number: 0
+  entry_points:
+    - mlconjug3=mlconjug3.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - click >=7.1
+    - colorama
+    - cython
+    - defusedxml
+    - joblib
+    - numpy
+    - python
+    - scikit-learn >=0.21.4
+    - scipy
+
+test:
+  imports:
+    - mlconjug3
+  commands:
+    - pip check
+    - mlconjug3 --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/SekouDiaoNlp/mlconjug3
+  summary: A Python library to conjugate French, English, Spanish, Italian, Portuguese and Romanian verbs using Machine Learning techniques.
+  license: MIT
+  license_file: LICENSE
+  description: "A Python library to conjugate verbs in French, English, Spanish, Italian, Portuguese and Romanian (more soon)\n    using Machine Learning techniques.\n\
```

Comment:
you can us YAML block variables to remove `\n` and just put line breaks.

Reply:
Thanks for the quick feedback @wolfv .
I will research  YAML block variables to remove \n and just put line breaks.

Reply:
Hi @wolfv I just made the necessary changes that you asked me to.
I used the YAML block variable to remove the extraneous `\n` .

Now the recipe should be ok. I will wait for the tests to pass, and will let you know.

Cheers,

Sekou.

Reply:
Hi @wolfv , all the checks have passed successfully.

Could you please kindly review the PR.

Thanks in advance.

Sekou.